iogt_users/tests.py

```diff
@@ -30,3 +30,22 @@ def test_user_locked_out_without_filling_registration_survey_form(self):
     def test_anonymous_user_can_browse_public_urls(self):
         response = self.client.get(self.home_page.url)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_admin_without_filling_registration_survey_form(self):
+        admin_user = AdminUserFactory(has_filled_registration_survey=False)
```

Comment:
I think this should be renamed so that it refers to 'superuser' instead of just admin. Something like, `test_superuser_should_not_complete_survey`.

Reply:
fixed it